website/src/components/canvas/sidebar/ResizeTextfields.jsx

```diff
@@ -0,0 +1,172 @@
+
+const styles = {
+    field: {
+        width: '30%'
+    },
+    smallIcon: {
+        height: 36,
+        width: 36
+    },
+    small: {
+        display: 'inline-block',
+        height: 60,
+        padding: 2,
+        width: 60
+    }
+};
+
+/**
+* Component of reisze textfields
+*/
+class ResizeTextfields extends React.Component {
+    /**
+   * Constructor for ResizeTextfields
+   * @param {Object} props The props for the ResizeTextfields.
+   */
+    constructor(props) {
+        super(props);
+        this.handleButtonAddHeight = this.handleButtonAddHeight.bind(this);
+        this.handleButtonAddWidth = this.handleButtonAddWidth.bind(this);
+        this.handleButtonSubHeight = this.handleButtonSubHeight.bind(this);
+        this.handleButtonSubWidth = this.handleButtonSubWidth.bind(this);
+        this.handlerTextfieldHeight = this.handlerTextfieldHeight.bind(this);
+        this.handlerTextfieldWidth = this.handlerTextfieldWidth.bind(this);
+    }
+    /**
+     * Handler that adds 1px to height
+     * @returns {void}
+     */
+    handleButtonAddHeight() {
+        let newSize = {
+            height: this.props.elements[this.props.targetedId].size.height + 1,
+            width: this.props.elements[this.props.targetedId].size.width
+        }
+        this.props.dispatch(EA.updateAndPersist(AC.UPDATE_SIZE, this.props.targetedId, newSize, this.props.currentCanvas));
+    }
+    /**
+     * Handler that adds 1px to width
+     * @returns {void}
+     */
+    handleButtonAddWidth() {
+        let newSize = {
+            height: this.props.elements[this.props.targetedId].size.height,
+            width: this.props.elements[this.props.targetedId].size.width + 1
+        }
+        this.props.dispatch(EA.updateAndPersist(AC.UPDATE_SIZE, this.props.targetedId, newSize, this.props.currentCanvas));
+    }
+    /**
+     * Handler that subtracts 1px to height
+     * @returns {void}
+     */
+    handleButtonSubHeight() {
+        let newSize = {
+            height: this.props.elements[this.props.targetedId].size.height - 1,
+            width: this.props.elements[this.props.targetedId].size.width
+        }
+        this.props.dispatch(EA.updateAndPersist(AC.UPDATE_SIZE, this.props.targetedId, newSize, this.props.currentCanvas));
+    }
+    /**
+     * Handler that subtracts 1px to width
+     * @returns {void}
+     */
+    handleButtonSubWidth() {
+        let newSize = {
+            height: this.props.elements[this.props.targetedId].size.height,
+            width: this.props.elements[this.props.targetedId].size.width - 1
+        }
+        this.props.dispatch(EA.updateAndPersist(AC.UPDATE_SIZE, this.props.targetedId, newSize, this.props.currentCanvas));
+    }
+    /**
+     * Handler for when the height field is changed.
+     * @param {Object} e  The event of changing the name.
+     * @param {String} newHeight The new name entered.
+     * @returns {void}
+     */
+    handlerTextfieldHeight(e, newHeight) {
```

Comment:
You could do a similar thing here if you wanted to where you have one function that generates these handlers or you could leave it because just 2 handlers will probably be about the same amount of code.